src/SDK/MonitoringApplianceEmulator/Converters/IntStringConverter.cs

```diff
@@ -0,0 +1,55 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IntStringConverter.cs" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//-----------------------------------------------------------------------
+
+namespace Microsoft.Azure.Monitoring.SmartDetectors.MonitoringApplianceEmulator.Converters
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Windows.Data;
+
+    /// <summary>
+    /// A converter class for handling safe conversion between string inputs to integers. All non integer inputs
+    /// will be converted to zeros
+    /// </summary>
+    public class IntStringConverter : IValueConverter
```

Comment:
>IntStringConverter [](start = 17, length = 18)

maybe IntToStringConverter

Reply:
nope - it's a two way conversion :)

---
In reply to: [231087930](https://github.com/Azure/SmartDetectorsMonitoringAppliance/pull/13#discussion_r231087930) [](ancestors = 231087930)